src/Selector/SelectorPrimitive.php

```diff
@@ -145,11 +145,21 @@ public static function extends(string $fqcn, bool $regex = false): ClassExtends
         return new ClassExtends($fqcn, $regex);
     }
 
+    /**
+     * @param class-string|non-empty-string $fqcn
+     *
+     * @deprecated use Selector::appliesAttribute()
```

Comment:
Note that PHPDocs recommend using `@see` to point to the new element:

```suggestion
     * @deprecated
     * @see Selector::appliesAttribute()
```

https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/deprecated.html#deprecated

> This tag MAY provide an additional description stating why the associated element is deprecated.
If the associated element is superseded by another, it is RECOMMENDED to add a [@see](https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/see.html) tag in the same PHPDoc pointing to the new element.